libraries/lib-music-information-retrieval/MirTypes.h

```diff
@@ -0,0 +1,64 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  MirTypes.h
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+#pragma once
+
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace MIR
+{
+enum class FalsePositiveTolerance
+{
+   Strict,
+   Lenient,
+};
+
+enum class TimeSignature
+{
+   TwoTwo,
+   FourFour,
+   ThreeFour,
+   SixEight,
+   _count
+};
+
+struct MusicalMeter
+{
+   const double bpm;
+   const std::optional<TimeSignature> timeSignature;
+};
+
+struct OnsetQuantization
+{
+   double error;
+   int lag;
+   int numDivisions;
```

Comment:
I got confused reviewing elsewhere -- this member actually is a number of tatums, not divisions